ui/src/experiments/list/search/components/ExperimentStatusFilter.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import { EuiComboBox, EuiFormRow, EuiHealth } from "@elastic/eui";
+
+const ExperimentStatusFilter = ({
+  value,
+  options,
+  onChange,
+}) => {
+  const renderOption = o => (<EuiHealth color={o.color}>{o.label}</EuiHealth>);
+  const selectedOptions = options.filter(o => o.value === value);
+  // Filter only fields recognized by the combobox.
+  const optionsFiltered = options.map(o => ({ value: o.value, label: o.label, color: o.color }));
+
+  return (
+    <EuiFormRow fullWidth label={"Experiment Status"}>
+      <EuiComboBox
+        placeholder="Select option"
+        options={optionsFiltered}
+        renderOption={renderOption}
+        selectedOptions={selectedOptions}
+        onChange={e => onChange(e.length > 0 ? e[0].value : undefined)}
+        isClearable={true}
+        data-test-subj={`experiment-status-combo-box`}
```

Comment:
Yes, either `` or "" will work here. But the `data-test-subj` is taken from the Eui sample code and it's used for running automated tests (`data-*` attributes are typically used to store some additional metadata; see https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes). Since we don't need it, I've removed its occurrences now.